exchanges/bitfinex/bitfinex_websocket.go

```diff
@@ -658,11 +658,19 @@ func (b *Bitfinex) wsHandleData(respRaw []byte) error {
 								b.Websocket.DataHandler <- offer
 							}
 						}
-					case strings.Contains(channelName, wsOrderNewRequest),
-						strings.Contains(channelName, wsOrderUpdateRequest),
```

Comment:
Prefer the assertion to be checked a little more explicitly. 
```diff
diff --git a/exchanges/bitfinex/bitfinex_websocket.go b/exchanges/bitfinex/bitfinex_websocket.go
index 49ddad0b3..31aa092b2 100644
--- a/exchanges/bitfinex/bitfinex_websocket.go
+++ b/exchanges/bitfinex/bitfinex_websocket.go
@@ -660,7 +660,12 @@ func (b *Bitfinex) wsHandleData(respRaw []byte) error {
                                                }
                                        case strings.Contains(channelName, wsOrderNewRequest):
                                                if data[2] != nil {
-                                                       if cid, ok := data[2].(float64); ok && cid > 0 {
+                                                       cid, ok := data[2].(float64)
+                                                       if !ok {
+                                                               return common.GetTypeAssertError("float64", data[2])
+                                                       }
+
+                                                       if cid > 0 {
                                                                if b.Websocket.Match.IncomingWithData(int64(cid), respRaw) {
                                                                        return nil
                                                                }
@@ -670,7 +675,12 @@ func (b *Bitfinex) wsHandleData(respRaw []byte) error {
                                        case strings.Contains(channelName, wsOrderUpdateRequest),
                                                strings.Contains(channelName, wsOrderCancelRequest):
                                                if data[0] != nil {
-                                                       if id, ok := data[0].(float64); ok && id > 0 {
+                                                       id, ok := data[0].(float64)
+                                                       if !ok {
+                                                               return common.GetTypeAssertError("float64", data[0])
+                                                       }
+
+                                                       if id > 0 {
                                                                if b.Websocket.Match.IncomingWithData(int64(id), respRaw) {
                                                                        return nil
                                                                }
```

Reply:
Sure. Can do. ~I didn't touch it since it was already there but that's no excuse. In fact, strikethrough.~
Though I'll still prefer to scope variables wherever possible as well. Saves so many lives.

Reply:
And Thanks 🙏

Reply:
Nit / Challenge: Feels like GetTypeAssertError could be common.AssertType and return the correct type OR an error.
Would require Generics, but we're requiring go 1.20 anyway so shrugapotamus 🤷 🦛